stats/Record.md

```diff
@@ -0,0 +1,73 @@
+# Record API Overview
+The stats library allows users to record metrics for their applications or third-party libraries.
+ The core data types used are:
+* `Measure`: describes the type of the individual values recorded by an application.
+* `Measurement`: describes a data point to be collected for a `Measure`.
+
+## Measure
+A `Measure` describes the type of the individual values/measurements recorded by an application. It
+includes information such as the type of measurement, the units of measurement and descriptive names
```

Comment:
nit: Measure also has a brief description.

Reply:
It is mentioned in the metadata section. Here is just an overview.